packages/components/src/components/Select/__tests__/Select.ct.tsx

```diff
@@ -316,3 +318,30 @@ test('should render custom clear slot', async ({ mount, page }) => {
     await expect(page.getByTestId(SLOT_CLEAR_TEST_ID)).toBeVisible();
     await expect(component).toContainText('Clear Slot');
 });
+
+test('should have max height equal to available space', async ({ mount, page }) => {
+    const component = await mount(
+        <Select aria-label="test" data-test-id={SELECT_TEST_ID} placeholder={PLACEHOLDER_TEXT}>
+            <Select.Slot name="menu">
+                <Select.Item data-test-id={ITEM_TEST_ID1} value="test1">
+                    {ITEM_TEXT1}
+                </Select.Item>
+                <Select.Item data-test-id={ITEM_TEST_ID2} value="test2">
+                    {ITEM_TEXT2}
+                </Select.Item>
+            </Select.Slot>
+        </Select>,
+    );
+
+    await expect(component).toBeVisible();
+    await component.click();
+
+    const dialog = page.getByTestId('fondue-select-menu');
+    await expect(dialog).toBeVisible();
+
+    const boundingBox = await dialog.boundingBox();
```

Comment:
I was thinking this could be an util function so we can easily test other components that need the same feature. Testing this generically, like testing the hook or the util function, seems cumbersome and leaves open the possibility that some quirk of a particular implementation of this will generate an unintended result that is not captured by a test